pkgs/tools/networking/smartdns/default.nix

```diff
@@ -0,0 +1,55 @@
+
+{ stdenv, fetchFromGitHub, openssl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "smartdns";
+  version = "30";
+  src = fetchFromGitHub {
+    owner = "pymumu";
+    repo = pname;
+    rev = "Release${version}";
+    sha256 = "1zj1ajvdnpmdal87j7fyn0xdd0ydjii1nvgwgacv2j311b6fdrk7";
+  };
+
+  buildInputs = [ openssl ];
+
+  # TODO: A patch is needed by now since the upstream doesn't follow the FHS guideline. It should not be the case in the future.
+  patches = [
```

Comment:
By convention, `patches` typically go after `src`. And there should be a blank line between `version` and `src`.

Reply:
Got it.